python/pyspark/sql/tests.py

```diff
@@ -5487,6 +5617,22 @@ def dummy_pandas_udf(df):
                                                  F.col('temp0.key') == F.col('temp1.key'))
         self.assertEquals(res.count(), 5)
 
+    def test_mixed_scalar_udfs_followed_by_grouby_apply(self):
+        # Test Pandas UDF and scalar Python UDF followed by groupby apply
+        from pyspark.sql.functions import udf, pandas_udf, PandasUDFType
+        import pandas as pd
```

Comment:
not a big deal at all really .. but I would swap the import order (thridparty, pyspark)